### PR TITLE
epinio and opni are no longer maintained.

### PR DESCRIPTION
**Pull request by l-mb**
Rancher Desktop is scaled down and should not be featured any more.

Signed-off-by: lmb@suse.com

### CODE REVIEW

**Comment by dirkmueller** [comment]
@l-mb it feels like rancher desktop recovered, should it still be removed?

File: db/featured.ts [hunk 1]
```diff
@@ -14,17 +14,6 @@ export const FEATURED_PROJECTS = [
...
-    color: '#0280BE',
-    logo: 'rancher-desktop.svg',
-    tag: 'development',
-    name: 'Rancher Desktop',
-    description: 'Kubernetes on your desktop.',
-    href: 'https://rancherdesktop.io/',
-    repositoryURL: 'https://github.com/rancher-sandbox/rd',
-    twitterURL: '',
-    documentationURL: '',
-  },
```
Comment (mattfarina) [inline db/featured.ts:27]: Rancher Desktop is still being actively developed and shipped. I don't think it needs to be removed.

File: db/featured.ts [hunk 2]
```diff
@@ -69,17 +58,6 @@ export const FEATURED_PROJECTS = [
...
-    color: '#004D93',
-    logo: 'epinio.svg',
-    tag: 'infrastructure',
-    name: 'Epinio',
-    description: 'An opinionated platform that runs on Kubernetes and takes you from code to URL in one step.',
-    href: 'https://epinio.io/',
-    repositoryURL: 'https://github.com/epinio',
-    twitterURL: '',
-    documentationURL: '',
-  },
```
Comment (mattfarina) [inline db/featured.ts:82]: I would hold off on Epinio, as well.

Comment (dirkmueller) [reply db/featured.ts:82]: it's not a suse project though anymore? so I think the change is ok.

File: db/featured.ts (hunk 1 above)
Comment (l-mb) [reply db/featured.ts]: I talked with Mike back then IIRC, the point was that Rancher Desktop was scaled down but not dead, yet would unlikely receive lots of development - hence why removing it from the featured projects rotation but not the full list of projects.
If we want to maintain the impression externally, I'm fine with that as well.

**Pull request merged**